### PR TITLE
bug fix for setting up route on linux with documentation clarifications

### PR DESCRIPTION
**Pull request by chrisbeardy**
duplicate of #97 due to git errors. Sorted out and rebased, new PR.

Corrected issue with setting up route to PLC using Linux.

### CODE REVIEW

**Comment by chrisbeardy** [comment]
I can now also confirm this works with TwinCat 2 and 3.

File: pyads/ads.py [hunk 1]
```diff
@@ -1,1153 +1,1153 @@
-"""Pythonic ADS functions.
-
-:author: Stefan Lehmann <stlm@posteo.de>
```
Comment (stlehmann) [inline pyads/ads.py:3]: You are replacing the whole file ads.py file. Please make sure only your changes for ads.py are in the PR.

**Comment by chrisbeardy** [comment]
sorry i'm struggling with yycharm and line endings here....

**Comment by chrisbeardy** [comment]
sorry having a nightmare today

**Pull request closed without merging**